app.js

```diff
@@ -2,22 +2,52 @@
 
 // List 3 expressions that surprised you or gave unexpected results, why?
 
+    // "2.5 * 'fish'" gave an unexpected result mainly because I had forgotten that NaN could come into play in a moment like this. 
+    // "'new york'[0]" was unexpected becuase I struggled to see 'new york' as an array, as opposed to a simple string.
+    // "'engineering'.indexOf('G')" surprised me because I initially expected an index of 2.
+
 // List 2 exprsssions you felt comfortable interpreting at a glance.
 
-// List 1 expression you are still 'shaky' on, why?
+    // (2 + -7) * 3
+    // 7 !== 7.1
 
+// List 1 expression you are still 'shaky' on, why?
 
+    // Any expression that invovles the "%" operator throws me off. I can predict most results, but I struggle to explain to myself the use cases for it.
```

Comment:
`%` is the modulus and looks for the remainder of a division operation. So if we have `10 % 2` we would get `0`. Here is a step-by-step breakdown of what `%` is doing:
```js
10 ÷ 2 = 5 // divide the dividend (10) by the divisor (2)
5 × 2 = 10 // take the result  (5) and multiply it by the divisor (2)
10 - 10 = 0 // take that result (10) and subtract it from the dividend (10)
```